lib/scarpe/radio.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class Radio < Scarpe::Widget
+    display_properties :group, :checked
+
+    def initialize(group = nil, checked = false, &block)
+      @group = group
+      @checked = checked
+      @block = block
+      super
+
+      bind_self_event("click") { click }
+      create_display_widget
+    end
+
+    def click(&block)
+      @block = block
+      self.checked = !checked?
+    end
+
+    def checked?
+      @checked ? true : false
+    end
+
+    def it_checked(value)
```

Comment:
I think this should be checked=.

Also, since checked is a display property, you should get @checked being assigned for free from self.checked=.

Reply:
okay will update